command/git.go

```diff
@@ -76,7 +76,7 @@ func buildHTTPCloneCommand(location *url.URL, branch, dir, token string) ([]stri
 		tokenFlag = fmt.Sprintf("-c 'credential.%s://%s.username=%s'", location.Scheme, location.Host, token)
 	}
 
-	clone := fmt.Sprintf("GIT_ASKPASS='true' git %s clone '%s' '%s'", tokenFlag, location.String(), dir)
+	clone := fmt.Sprintf("git %s clone '%s' '%s'", tokenFlag, location.String(), dir)
```

Comment:
The docs for GIT_ASKPASS suggest that it is passed a binary that knows how to ask for passwords, and that potentially passing true means a noop? I can't actually figure out what passing "true" might mean from the docs. What are you trying to do here?